catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/QueryApplication.java

```diff
@@ -244,6 +300,72 @@ private CqlQueryResponse executeCqlQuery(CqlRequest cqlRequest)
         actionRegistry);
   }
 
+  private void attachFileWithTransformer(
```

Comment:
not sure if i should break down this helper method even further. for each helper method i may need to pass request, response with it